lib/board.rb

```diff
@@ -36,14 +36,22 @@ def winning_player_exists?(player1_mark, player2_mark)
   end
 
   def complete?
-    available_squares.zero?
+    available_squares_quantity.zero?
   end
 
   def clear_squares
     @squares = [1, 2, 3, 4, 5, 6, 7, 8, 9]
   end
 
-  def available_squares
+  def available_squares_quantity
     @squares.count { |square| square.is_a? Integer }
```

Comment:
There is duplication here - ideally this should use the `available_squares` method and then just call `size`